### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by tta2yta**
In the final touches of the project, I have used CSS to improve the look and feel of the application.

Kindly review my code and give me a review.
Thanks in Advance

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -58,3 +64,29 @@ input[type="submit"] {
...
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    width: 700px;
+    margin: auto;
+    margin-top: 100px;
+  }
+}
```
Comment (Dipeshtwis) [inline src/style.css]: - [ ] The submit button is not in the right position and the layout looks little different. Please fix this.

### Your design 

Screenshot from 2021-07-29 04-19-34


### Wireframe design 

Screenshot from 2021-07-29 04-20-17

Comment (Dipeshtwis) [inline src/style.css]: [OPTIONAL]

- I would recommend to use whole window as container instead of making a little one which left so much gap in the UI.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @tta2yta, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: README file looks professional and contains all that's needed for running the app.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bellom — APPROVED** [review]
## Hi @tta2yta  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**